modules/tinymce/src/plugins/link/test/ts/browser/DialogOpenTest.ts

```diff
@@ -66,4 +68,76 @@ describe('browser.tinymce.plugins.link.DialogOpenTest', () => {
     TinyContentActions.keystroke(editor, 'K'.charCodeAt(0), metaKey);
     await pDialogCheck(editor);
   });
+
+  const pTestDialogOnSelection = async (
+    startPath: number[],
+    soffset: number,
+    finishPath: number[],
+    foffset: number,
+    result: string
+  ) => {
+    const editor = hook.editor();
+    editor.setContent('<p>aaa bbb ccc</p><p>ddd</p>');
+    TinySelections.setSelection.apply(null, [ editor, startPath, soffset, finishPath, foffset ]);
```

Comment:
I assume that this is part of some refactor change that was made but not fully restored?

```suggestion
    TinySelections.setSelection(editor, startPath, soffset, finishPath, foffset);
```

Reply:
Yeah, forgot to change this. Thanks for pointing it out! Fixed [here](https://github.com/tinymce/tinymce/pull/8386/commits/86b37c56b64208ea0065ad8183fb75a356241919).